src/http.js

```diff
@@ -33,7 +32,7 @@ export default class HTTP {
      * The eventt emitter instance.
      * @type {EventEmitter}
      */
-    this.events = options.events || new EventEmitter();
+    this.events = options.events;
```

Comment:
Note that the same as for `Api` applies here, `events` should become a required argument.